lib/health_checks/sidekiq.rb

```diff
@@ -40,28 +40,32 @@ def start(mongoid_databases, redis_configs)
       end
     end
 
-    def create_clients(mongoid_databases)
-      mongoid_databases.map do |db|
+    def create_clients(mongo_databases)
+      mongo_databases.map do |db|
         client_name = "#{db[:name]}_sidekiq_health_check"
         client = MongoidCustomClient.create(client_name, db[:hosts])
 
         { db_name: db[:name], instance: client }
       end
     end
 
-    def check_mongoid_clients(clients)
-      db_name = ''
+    def check_mongo_connections(clients)
       clients.each do |client|
-        db_name = client[:db_name]
-        client[:instance].command(dbStats: 1).first['db']
+        begin
+          client[:instance].command(dbStats: 1).first['db']
+        rescue => e
+          raise "#{e}, Database Name: #{client[:db_name]}"
```

Comment:
You specify the name here and then when you log the error you specify the name again. Either do it here or when logging. For Sidekiq you only do it in one place - "here"